hl7-mllp/2.1/modules/ROOT/edits/hl7-mllp-connector-examples.adoc

```diff
@@ -0,0 +1,136 @@
+= HL7 EDI - Examples - Mule 4
+
+Use these examples to read and write HL7 messages over the MLLP protocol.
+
+== Example: HL7 Studio
```

Comment:
This title is misleading: this is not an example but rather prerequisites to following the two examples provided.

Reply:
Fixed.